DESCRIPTION

```diff
@@ -1,15 +1,19 @@
 Package: christmas
-Title: R functions to help with christmas admin
-Version: 0.0.0.9001
-Authors@R: c(
-    person(given = "Nicola", family = "Rennie", role = c("aut", "cre", "cph"),
-    email = "nrennie35@gmail.com", comment = c(ORCID = "0000-0003-4797-557X")))
-Description: R functions to help with christmas admin.
-Imports:
-  ggforce,
-  ggplot2,
-  sf
+Title: R functions to help with Christmas admin
+Version: 0.0.0.9002
+Authors@R: 
+    person("Nicola", "Rennie", , "nrennie35@gmail.com", role = c("aut", "cre", "cph"),
```

Comment:
The formatting of Authors@R has gone a bit weird? Keep the args named, and avoid double comma.

Reply:
Also, you've done quite a lot of work on this so feel like you should add yourself as an author 💯

Reply:
strange... `usethis::use_tidy_description()` did that. I have undone it [edb24fc](https://github.com/nrennie/christmas/pull/1/commits/edb24fc9486ff2a1eee4f327cc38f48ce044b986)

Reply:
thanks, but you did all the work really.